apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -303,7 +330,9 @@ const ApplicationReview = ({
                     {intl.formatMessage({
                       defaultMessage:
                         "It looks like you haven't added any experiences to your résumé yet.",
-                      id: "bOLaWz",
+                      id: "ifzI0r",
+                      description:
+                        "Null state message for education requirement section of the application review page.",
```

Comment:
Consider changing to something along the lines of:

Null state message for experiences section of the application review page.

Reply:
Fixed by [a972a4e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/a972a4ed7e18c3b9fb1392ec7b859fbc5815cc4c)